test/fixtures/node_tags.yml

```diff
@@ -10,6 +10,12 @@ awesome:
   nid: 1
   date: <%= DateTime.now.to_i %>
 
+ cool:
```

Comment:
I think this line needs zero spaces before it -- each double-space is counted as an indent and affects YAML formatting. Thanks!

Reply:
@jywarren done ! :)

Reply:
Hmm, it's still showing YAML errors. Could there be more?